docs/integration/revoke.adoc

```diff
@@ -54,7 +54,7 @@ The body includes a JSON object:
 |===============================================================================
 |# |Name |Type |Description
 |1 |accountId |String |The ID of the users account
-|2 |endpointIds |Array of String |An Array of endpointIDs, that shall be deleted.
+|2 |endpointIds |Array of String |An Array of endpointIDs that shall be deleted.
```

Comment:
second endpointIDs should be endpointIds

Reply:
will fix